service/src/main/java/com/myhome/controllers/dto/PaymentDto.java

```diff
@@ -26,12 +26,14 @@
 @Getter
 @Setter
 public class PaymentDto {
+  private Long id;
```

Comment:
Adding this `id` here helps updating the payment entity in the `PaymentSDJpaService`